cilium/datadog_checks/cilium/data/conf.yaml.example

```diff
@@ -0,0 +1,29 @@
+init_config:
+
+instances:
+
+    ## @param agent_endpoint - string - required
+    ## The URL where your application metrics are exposed by Prometheus.
+    ## By default, the Cilium integration collects `cilium-agent` metrics.
+    ## One of agent_endpoint or operator_endpoint must be provided.
+    #
+  - agent_endpoint: http://localhost:9090/metrics
+
+    ## @param operator_endpoint - string - optional
+    ## Provide instead of `agent_endpoint` to collect `cilium-operator` metrics.
+    ## Cilium operator metrics are exposed on port 6942.
+    #
+    # operator_endpoint: http://localhost:6942/metrics
+
+    ## @param tags - list of key:value elements - optional
+    ## List of tags to attach to every metrics, events and service checks emitted by this integration.
+    ## Learn more about tagging: https://docs.datadoghq.com/tagging/
+    #
+    # tags:
+    #   - <KEY_1>:<VALUE_1>
+    #   - <KEY_2>:<VALUE_2>
+
+    ## @param timeout - integer - optional - default: 10
+    ## Overrides the default timeout value in second
+    #
+    # timeout: 10
```

Comment:
There's much more configuration parameter in the OpenMetrics base class. Let's at least add the relevant one about SSL and see if we should add others.